packages/manager/modules/pci/src/components/project/flavors-list/translations/Messages_fr_FR.json

```
@@ -13,6 +13,10 @@
   "pci_project_flavors_spec_core_details": "{{vcores}} vCores ({{frequency}} GHz)",
   "pci_project_flavors_spec_disk": "Stockage :",
   "pci_project_flavors_spec_bandwidth": "Bande passante :",
+  "pci_project_flavors_spec_addons": "Addon :",
+  "pci_project_flavors_spec_addons_detail_single": "{{ addons }} card",
+  "pci_project_flavors_spec_addons_detail_multi": "{{ addons }} cards",
+  "pci_project_flavors_spec_model": "Model :",
```

is this the content provided ? 
```suggestion
  "pci_project_flavors_spec_model": "Modèle :",
```

---

No... This was written by me and I didn't know the corresponding correct french word. Thanks for providing it :)